### PR TITLE
Cqlmapper support

### PR DESCRIPTION
**Pull request by pacejackson**

### CODE REVIEW

File: baseplate/context/cqlmapper.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from cqlmapper.connection import Connection as CQLConnection
```
Comment (spladug) [inline baseplate/context/cqlmapper.py]: I always get kinda confused by these import-and-rename things. How about just doing `import cqlmapper.connection` and using `return cqlmapper.connection.Connection` if you want it scoped or just go straight up unmangled `Connection`.

File: baseplate/context/cqlmapper.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+from .cassandra import CassandraContextFactory
+
+
+class CQLMapperContextFactory(CassandraContextFactory):
```
Comment (spladug) [inline baseplate/context/cqlmapper.py]: would you be OK just putting this in the `cassandra` module so it's all in one place? that might mean putting the import in the method call so it's not required for non-cqlmapper clients, but that's OK.

**Comment by pacejackson** [comment]
screen shot 2017-04-27 at 5 27 12 pm

**Comment by pacejackson** [comment]
:haircut_man:

**Pull request merged**